agent/taskresource/credentialspec/credentialspec_windows.go

```diff
@@ -18,6 +18,8 @@ package credentialspec
 import (
```

Comment:
Can you add description to the pr to describe why these code changes are needed? This can help our team members who don't have context to understand your pr.

Reply:
Sure thing.